### PR TITLE
Improvement in hindi translation

### PR DESCRIPTION
**Pull request by shivammaindola**
Corrected some hindi words.

### CODE REVIEW

**Comment by shivammaindola** [comment]
@cpg can you please review this PR?

File: src/main/res/values-hi-rIN/strings.xml [hunk 1]
```diff
@@ -115,7 +115,7 @@
     <string name="message_duplicate_file_upload_body">फ़ाइल% 1 $ s पहले से मौजूद है। क्या आप वाकई मौजूदा फ़ाइल को बदलना चाहते हैं?</string>
     <string name="notification_upload_title">नई छवि अपलोड कर रहा है</string>
     <string name="notification_upload_message">फ़ाइल% 1 $ s</string>
-    <string name="message_upload_success">अपलोड पूरा हुआ</string>
+    <string name="message_upload_success">अपलोड पूरा हूवा</string>
```
Comment (balaji-ramavathu) [inline src/main/res/values-hi-rIN/strings.xml]: हुआ is the correct word here.

File: src/main/res/values-hi-rIN/strings.xml [hunk 2]
```diff
@@ -152,24 +152,24 @@
...
-    <string name="intro_desc_phone_1">एक एंड्रॉइड ऐप जो आपको अपने अम्मी होम सर्वर में फाइलों तक पहुंचने देता है।</string>
+    <string name="intro_tv_1">एंड्राइड टीवी के लिए अमाही में आपका स्वागत है</string>
+    <string name="intro_phone_1">एंड्राइड के लिए अमाही में आपका स्वागत है।</string>
+    <string name="intro_desc_phone_1">एक एंड्रॉइड ऐप जो आपको अपने अमाही होम सर्वर में फाइलों तक पहुंचने देता है।</string>
     <string name="intro_title_2">अपने HDA तक पहुँचें</string>
     <string name="intro_desc_2">अमाही आपको अपने वीडियो चलाने, अपनी तस्वीरों को देखने, अपने संगीत को सुनने और बहुत कुछ करने देता है!</string>
     <string name="intro_title_3">अपनी तस्वीरें ब्राउज़ करें</string>
-    <string name="intro_desc_tv_3">Browse your photo library in your TV, fast and at amazing resolution.</string>
-    <string name="intro_desc_phone_3">अपने फोटो लाइब्रेरी को आसानी से ब्राउज़ करें, सीधे अपने फोन से फोटो अपलोड करें।</string>
+    <string name="intro_desc_tv_3">अपने टीवी, तेज और अद्भुत संकल्प में अपनी फोटो लाइब्रेरी ब्राउज़ करें</string>
```
Comment (balaji-ramavathu) [inline src/main/res/values-hi-rIN/strings.xml]: The meaning here is not the same as in English. Please make it proper.

File: src/main/res/values-hi-rIN/strings.xml [hunk 3]
```diff
@@ -124,10 +124,10 @@
     <string name="notification_offline_download_complete">फ़ाइल ऑफ़लाइन उपलब्ध है</string>
     <string name="notification_offline_download_paused">इंटरनेट की प्रतीक्षा में</string>
     <string name="notification_offline_download_error">फ़ाइल को ऑफ़लाइन उपलब्ध कराते समय त्रुटि</string>
-    <string name="share_subject">मेरे Amahi होम सर्वर की जाँच करें!</string>
-    <string name="share_message">"मैं अपनी सभी फाइलों को स्टोर करने, बैकअप लेने और स्ट्रीमिंग के लिए अम्मी होम सर्वर का उपयोग करता हूं। इसकी जांच - पड़ताल करें! <a href="https://www.amahi.org/"> https://www.amahi.org/ </a>  "</string>
-    <string name="title_version_settings">Android के लिए अमही</string>
-    <string name="message_no_search_result">कोई खोज परिणाम नहीं मिला।</string>
+    <string name="share_subject">मेरे अमाही होम सर्वर की जाँच करें!</string>
+    <string name="share_message">मैं अपनी सभी फाइलों को स्टोर करने, बैकअप लेने और स्ट्रीमिंग के लिए अम्मी होम सर्वर का उपयोग करता हूं। इसकी जांच - पड़ताल करें! <a href="https://www.amahi.org/"> https://www.amahi.org/ </a> </string>
+    <string name="title_version_settings">एंड्राइड के लिए अमही</string>
```
Comment (csoni111) [inline src/main/res/values-hi-rIN/strings.xml]: अमही -> अमाही

**Comment by cpg** [comment]
I don't know if the suggestions are merged or not. @csoni111 please squash and merge if they are.

**Pull request merged**